config/manager/manager.yaml

```diff
@@ -113,10 +113,9 @@ spec:
           # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
           resources:
             limits:
-              cpu: 500m
-              memory: 128Mi
+              memory: 2Gi
```

Comment:
we have taken of the cpu limit and increased the memory limit and the requests too. can you add a note about why we need this?

Reply:
we discussed about the increase over chat, resolving the conversation